modules/37_industry/subsectors/equations.gms

```diff
@@ -212,6 +212,35 @@ q37_costCESmarkup(t,regi,in)$(ppfen_industry_dyn37(in))..
   * (vm_cesIO(t,regi,in) + pm_cesdata(t,regi,in,"offset_quantity"))
 ;
 
+***------------------------------------------------------
+*' Near-term limits of switching to biomass use in industry subsectors
+***------------------------------------------------------
+q37_BioLimitSubsec(t,regi,entyFe,emiMkt)..
+  sum( se2fe(entySeBio,entyFe,te),
+    vm_demFEsector(t,regi,entySeBio,entyFE,"indst",emiMkt)
+  )
+  =l=
+  sum(secInd37_emiMkt(secInd37,emiMkt),
+      p37_BioShareMaxSubsec(t,regi,entyFe,secInd37)
+      * sum( (secInd37_2_pf(secInd37,in),
+              fe2ppfen37(entyFe,in)),
+          vm_cesIO(t,regi,in))
+  )
+;
```

Comment:
This will break under PBS, unless `vm_outflowPrc` is taken into account, like in `q37_demFeIndst`.  (`q37_demFeIndst_intermediate` on the current merge target, which is outdated.)